src/number_theory/number_field.lean

```diff
@@ -4,23 +4,27 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Ashvni Narayanan, Anne Baanen
 -/
 
-import algebra.field.basic
-import data.rat.basic
-import ring_theory.algebraic
 import ring_theory.dedekind_domain.integral_closure
-import ring_theory.integral_closure
-import ring_theory.polynomial.rational_root
+import algebra.char_p.algebra
+import analysis.complex.polynomial
 
 /-!
 # Number fields
-This file defines a number field and the ring of integers corresponding to it.
+This file defines a number field, the ring of integers corresponding to it and includes some
+basic facts about the embeddings into `ℂ` of a number field.
 
 ## Main definitions
  - `number_field` defines a number field as a field which has characteristic zero and is finite
     dimensional over ℚ.
  - `ring_of_integers` defines the ring of integers (or number ring) corresponding to a number field
     as the integral closure of ℤ in the number field.
 
+## Main Results
+ - `lift`: for `L/K` an extension of number fields, any embedding of a number field `K`
+    can be lifted to an embedding of `L`.
+ - `eq_roots`: let `x ∈ K` with `K` number field, then the images of `x` by the embeddings
+    of `K` are exactly the roots in `ℂ` of the minimal polynomial of `x` over `ℚ`.
```

Comment:
I suggest to replace `ℂ` by any algebraically closed field of characteristic `0`. (This could make useless my above comment.)

Reply:
Yes, this is a very good idea and probably easy to do. I'll start with that. I'll have a look at your other remarks soon.

Reply:
Results now hold for any algebraic closed (char. 0) field